src/renderer/components/VulnerableForm.tsx

```
@@ -0,0 +1,130 @@
+
+  // ISSUE: No input validation
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    
+    // ISSUE: Sending sensitive data in URL
+    const response = await axios.get(
+      `http://api.example.com/submit?name=${formData.name}&email=${formData.email}&message=${formData.message}`
+    );
+    
+    // ISSUE: eval on response data
+    eval(`var result = ${JSON.stringify(response.data)}`);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)